lib/semantics/semantics.h

```diff
@@ -193,5 +193,8 @@ struct BaseChecker {
   template<typename N> void Enter(const N &) {}
   template<typename N> void Leave(const N &) {}
 };
+
+// Static declaration checks
+void CheckDeclarations(SemanticsContext &);
```

Comment:
In my opinion this should be in `check-declarations.h`, even though it means creating an extra file.

Reply:
No problem.